crates/flowctl/Cargo.toml

```diff
@@ -15,17 +15,21 @@ assemble = { path = "../assemble" }
 connector-init = { path = "../connector-init" }
 derive-typescript = { path = "../derive-typescript" }
 doc = { path = "../doc" }
+json = { path = "../json" }
 journal-client = { path = "../journal-client" }
 labels = { path = "../labels" }
 models = { path = "../models" }
 ops = { path = "../ops" }
 proto-flow = { path = "../proto-flow" }
+proto-grpc = { path = "../proto-grpc", features = ["capture_client"] }
 proto-gazette = { path = "../proto-gazette" }
 runtime = { path = "../runtime" }
 schema-inference = { path = "../schema-inference" }
 sources = { path = "../sources" }
-tables = { path = "../tables" }
+tables = { path = "../tables", features = ["persist"] }
```

Comment:
You enable `rusqlite` and persistence, but I don't actually see them used. Remove?